zendesk/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,61 @@
+= Zendesk Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Zendesk enables you to connect to the Zendesk Support API.
+
+For information about compatibility and fixed issues, see the Zendesk Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Zendesk Connector's API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to:
+
+* The Zendesk target resource
+* Anypoint Platform
+* Anypoint Studio version 4.2.1 or later
+
+== Common Use Cases For the Connector
+
+Common use cases for Zendesk Connector include the following:
+
+* Integrating Salesforce with Zendesk to automate actions
++
+For example, when a new user is created or updated in Zendesk, a corresponding record is created or updated in Salesforce.
++
+* Integrating Stripe with Zendesk to automate actions
++
+For example, when a user is created in Zendesk based on a customer request, the user is created in Stripe to track future purchases and subscriptions.
++
+* Communicating with Slack
++
+For example, when a company receives a new attachment with a photo of an ID in Slack, a ticket opens in Zendesk so that an employee can verify this information.
```

Comment:
change this to: "send case data from zendesk as a message in slack to drive effective collaboration and resolution."